src/main/resources/JavadocParser.g4

```diff
@@ -110,13 +110,19 @@ blockTag
     | deprecatedTag
     | returnTag
     | parameterTag
+    | throwsTag
+    | exceptionTag
     | customBlockTag
     ;
 
 authorTag: AUTHOR_LITERAL description;
 
 deprecatedTag: DEPRECATED_LITERAL description;
 
+throwsTag: THROWS EXCEPTION_NAME description?;
+
+exceptionTag: EXCEPTION EXCEPTION_NAME description?;
```

Comment:
> EXCEPTION_NAME

https://github.com/openjdk/jdk/blob/8ea544c33fc502208577249fb83544f8d876bc17/src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java#L2411
Maybe I am not understanding their code, but they reference this a reference. Think they did this just to not have to rewrite some similar code?